toponetx/utils/normalization.py

```diff
@@ -0,0 +1,383 @@
+"""Normalize of adjacency, boundary matrices of complexes."""
+
+import numpy as np
+import scipy.sparse.linalg as spl
+from numpy import ndarray
+from numpy.linalg import pinv
+from scipy.sparse import csr_matrix, diags, identity
+
+__all__ = [
+    "compute_laplacian_normalized_matrix",
+    "compute_x_laplacian_normalized_matrix",
+    "compute_kipf_adjacency_normalized_matrix",
+    "compute_xu_asymmetric_normalized_matrix",
+    "compute_bunch_normalized_matrices",
+    "_compute_B1_normalized_matrix",
+    "_compute_B1T_normalized_matrix",
+    "_compute_B2_normalized_matrix",
+    "_compute_B2T_normalized_matrix",
+    "_compute_D1",
+    "_compute_D2",
+    "_compute_D3",
+    "_compute_D5",
```

Comment:
Remove the internal functions (starting with an underscore) from `__all__`, please :)